### PR TITLE
1

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by wlisik** [comment]
ok

File: src/org/ft/pkg/TestClass.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package org.ft.pkg;
```
Comment (Bombe) [inline src/org/ft/pkg/TestClass.java]: What the…

File: src/plugins/Freetalk/Board.java [hunk 2]
```diff
@@ -26,7 +26,8 @@
  * Represents a forum / newsgroups / discussion board in Freetalk. Boards are created by the <code>MessageManager</code> on demand, you do
  * not need to manually create them. The <code>MessageManager</code> takes care of anything related to boards, to someone who just wants to
  * write a user interface this class can be considered as read-only.
- *
+ * 
+ * {@docRoot}
```
Comment (Bombe) [inline src/plugins/Freetalk/Board.java]: ?

**Comment by Bombe** [comment]
Unless you can provide good reasons for deleting tests, removing the build script, adding some Eclipse configuration file and adding empty classes, this will be only go one way: down the drain.

**Comment by xor-freenet** [comment]
Closing in favor of https://github.com/freenet/plugin-Freetalk/pull/21 which contains all commits of this PR.

**Pull request closed without merging**